OSDOCS-16871-3: SCALE-1 Core Scalability Planning and Resource Manage (#106034)
openshift/openshift-docs: modules/maintaining-bare-metal-hosts.adoc

[id="maintaining-bare-metal-hosts_{context}"]
= Maintaining bare metal hosts

[role="_abstract"]

> To ensure your cluster inventory accurately reflects your physical infrastructure, maintain the details of the bare-metal hosts. You can update these host configurations directly from the {product-title} web console.

Reviewer: Could combine these imo
```suggestion
To ensure your cluster inventory accurately reflects your physical infrastructure, maintain the details of the bare-metal host configurations by using the {product-title} web console.
```
